### PR TITLE
[Jetnews] Showcase PreviewParameterProvider usage.

### PR DESCRIPTION
**Pull request by JolandaVerhoef**

### CODE REVIEW

File: JetNews/app/src/debug/java/com/example/jetnews/PostPreviewParameterProvider.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import com.example.jetnews.data.posts.impl.post1
+import com.example.jetnews.data.posts.impl.post2
+import com.example.jetnews.data.posts.impl.post3
+import com.example.jetnews.data.posts.impl.post4
+import com.example.jetnews.data.posts.impl.post5
+import com.example.jetnews.model.Post
+
+class PostPreviewParameterProvider : PreviewParameterProvider<Post> {
```
Comment (manuelvicnt) [inline JetNews/app/src/debug/java/com/example/jetnews/PostPreviewParameterProvider.kt:27]: Add comments about what this does and information about the feature, and also maybe link to the tooling docs?

Comment (manuelvicnt) [reply JetNews/app/src/debug/java/com/example/jetnews/PostPreviewParameterProvider.kt:27]: what's the best practice for this? Should the file be moved to something like a `previews` package?

Comment (JolandaVerhoef) [reply JetNews/app/src/debug/java/com/example/jetnews/PostPreviewParameterProvider.kt]: I moved the class to a `model` package. That way, the preview class is in the same directory as the Model class that it creates. This to me seems a logical organization for it.

I added a comment explaining the usage. There's no Tooling doc to point to (yet), as there's no documentation of this feature anywhere. I will add that to the Tooling doc when I find the time for it!

Comment (manuelvicnt) [reply JetNews/app/src/debug/java/com/example/jetnews/PostPreviewParameterProvider.kt]: SG! Thank you

**Review by manuelvicnt — APPROVED** [review]

**Pull request merged**